src/lib/ui/ClimateSettingForm/ClimateSettingScheduleDeviceSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import { isThermostatDevice } from 'seamapi'
+
+import { DeviceTable, type UseDevicesData } from 'lib/index.js'
+
+interface ClimateSettingScheduleDeviceSelectProps {
+  onSelect: (deviceId: string) => void
+}
+
+export function ClimateSettingScheduleDeviceSelect({
+  onSelect,
+}: ClimateSettingScheduleDeviceSelectProps): JSX.Element {
+  const deviceFilter = (
```

Comment:
This should be moved outside the render body.

Reply:
https://github.com/seamapi/react/pull/440/commits/9ae3a55c464f681bf4066f67550d1c70d1d7c49c